### PR TITLE
Add an option to show or hide the menu

### PR DESCRIPTION
**Pull request by prototux**
This add an checkbox (in the menu itself, with key F5 as well) "Show Menu", enabled/checked by default.

This option is useful in case the pinball is the only item displayed on the screen, in my case, as a fun addition on my mini arcade machine (https://i.imgur.com/nz3u48A.jpg) :slightly_smiling_face: 

Didn't tested it yet, i'll update the PR after it has been tested

### CODE REVIEW

**Comment by Sunlitspace542** [comment]
I've tested this and can confirm it works, however, the F5 Key is already taken in the code, and the menu show/hide key wasn't added to the keyboard switch statement in options.cpp, after fixing these issues (set show ui to F9), it works.

**Comment by prototux** [comment]
> I've tested this and can confirm it works, however, the F5 Key is already taken in the code, and the menu show/hide key wasn't added to the keyboard switch statement in options.cpp, after fixing these issues (set show ui to F9 in my case), it works.

Thanks for the feedback! i've fixed both of these issues :slightly_smiling_face: didn't saw the F5 was already taken, sorry.

File: SpaceCadetPinball/winmain.cpp [hunk 1]
```diff
@@ -573,6 +584,9 @@ int winmain::event_handler(const SDL_Event* event)
 				pause();
 			options::keyboard();
 			break;
+		case SDLK_F0:
```
Comment (k4zmu2a) [inline SpaceCadetPinball/winmain.cpp:587]: This should be SDLK_F9, F0 is not defined.

**Comment by k4zmu2a** [comment]
Other than F9 issue, everything else looks good to me, I will accept this.

**Comment by prototux** [comment]
> Other than F9 issue, everything else looks good to me, I will accept this.

And that's why coding at 3AM is a bad idea :sweat_smile: it should be fixed if i didn't messed up the history with the amend.

**Comment by k4zmu2a** [comment]
Thank you, merged.

**Pull request merged**